package.json

```diff
@@ -5,10 +5,11 @@
   "main": "server.js",
   "scripts": {
     "build-css": "./css-build",
+    "coverage-clear": "rm -Rf coverage",
```

Comment:
Since it's a clean file system `coverage` won't exist and the command will do nothing and exit with a 0.